examples/multisource_adapt/configs/Office2A-M3SDA.yaml

```diff
@@ -0,0 +1,13 @@
+DAN:
+  METHOD: "M3SDA"
+DATASET:
+  NAME: "office_caltech"
+  NUM_CLASSES: 10
+  ROOT: "../data/office"
+  TARGET: "amazon"
```

Comment:
I guess you want to say that the sources are implied to be the other three domains, add a comment maybe e.g. after target? Do you support 2 sources?